homeassistant/components/abode/config_flow.py

```diff
@@ -26,48 +27,151 @@ def __init__(self):
             vol.Required(CONF_PASSWORD): str,
         }
 
+        self.mfa_data_schema = {
+            vol.Required(CONF_MFA): str,
+        }
+
+        self._username = None
+        self._password = None
+        self._polling = None
+        self._cache = None
+
+    async def _async_abode_login(self, step_id):
+        """Handle login with Abode."""
+        self._cache = self.hass.config.path(DEFAULT_CACHEDB)
+        errors = {}
+
+        try:
+            abode = Abode(
```

Comment:
I don't need to create an instance of `Abode` here. Only required for the mfa step since the `login` method of `Abode` is only accessible that way.